viaq-openshift/skeleton.json

```diff
@@ -1,5 +1,7 @@
 {
-    "aliases": {},
+    "aliases": {
+        "project.{index}" : {}
```

Comment:
If this is working, go ahead and merge it, and update https://github.com/openshift/origin-aggregated-logging/pull/222